### ISSUE TITLE
Console Error

### ISSUE THREAD

**Issue by andrew-kaslick** [webtoolbox/craftcms-plugin#7]
Hi,

We're using the Website Toolbox Forum plugin version `1.4.0` on Craft `3.7.38`. Running any Craft CLI commands from the console results in the below error. It looks like there may need to be some checks added using `Craft::$app->getRequest()->getIsConsoleRequest()` since the request session is not available from console requests.
```
Exception 'craft\errors\MissingComponentException' with message 'Session does not exist in a console request.'

in /var/www/vendor/craftcms/cms/src/console/Application.php:129

Stack trace:
#0 /var/www/vendor/websitetoolbox/websitetoolboxcommunity/src/Websitetoolboxcommunity.php(73): craft\console\Application->getSession()
#1 /var/www/vendor/yiisoft/yii2/base/BaseObject.php(109): websitetoolbox\websitetoolboxcommunity\Websitetoolboxcommunity->init()
#2 /var/www/vendor/yiisoft/yii2/base/Module.php(161): yii\base\BaseObject->__construct()
#3 /var/www/vendor/craftcms/cms/src/base/Plugin.php(115): yii\base\Module->__construct()
#4 [internal function]: craft\base\Plugin->__construct()
#5 /var/www/vendor/yiisoft/yii2/di/Container.php(419): ReflectionClass->newInstanceArgs()
#6 /var/www/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build()
#7 /var/www/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get()
#8 /var/www/vendor/craftcms/cms/src/Craft.php(61): yii\BaseYii::createObject()
#9 /var/www/vendor/craftcms/cms/src/services/Plugins.php(948): Craft::createObject()
#10 /var/www/vendor/craftcms/cms/src/services/Plugins.php(232): craft\services\Plugins->createPlugin()
#11 /var/www/vendor/craftcms/cms/src/base/ApplicationTrait.php(1487): craft\services\Plugins->loadPlugins()
#12 /var/www/vendor/craftcms/cms/src/console/Application.php(48): craft\console\Application->_postInit()
#13 /var/www/vendor/yiisoft/yii2/base/BaseObject.php(109): craft\console\Application->init()
#14 /var/www/vendor/yiisoft/yii2/base/Application.php(204): yii\base\BaseObject->__construct()
#15 /var/www/vendor/yiisoft/yii2/console/Application.php(89): yii\base\Application->__construct()
#16 [internal function]: yii\console\Application->__construct()
#17 /var/www/vendor/yiisoft/yii2/di/Container.php(419): ReflectionClass->newInstanceArgs()
#18 /var/www/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build()
#19 /var/www/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get()
#20 /var/www/vendor/craftcms/cms/src/Craft.php(61): yii\BaseYii::createObject()
#21 /var/www/vendor/craftcms/cms/bootstrap/bootstrap.php(247): Craft::createObject()
#22 /var/www/vendor/craftcms/cms/bootstrap/console.php(51): require('...')
#23 /var/www/craft(12): require('...')
#24 {main}
```

**Comment by webtoolbox** [comment]
Hi Andrew! Thanks for reporting the issue, and for your suggestion regarding how to fix it. I apologize for the inconvenience. Our development team has made changes to fix the issue and those changes are currently undergoing QA testing. We'll update you soon once the changes are published.

**Comment by webtoolbox** [comment]
Thanks for your patience while we worked on this. This is fixed in version 1.4.1 of the plugin.
